services/prometheus.nix

```diff
@@ -0,0 +1,58 @@
+{ config, lib, ... }:
+let
+  # We Should be able to generate this
```

Comment:
I've been working on a new way to structure the repo such that we can resolve other nodes in the fleet and know what is deployed to them, instead of it being defined by their imports.